appengine/standard/i18n/i18n_utils.py

```diff
@@ -19,7 +19,7 @@
 The idea of this example, especially for how to translate strings in
 Javascript is originally from an implementation of Django i18n.
 """
-
+from __future__ import print_function
```

Comment:
I'm not sure you need this import.

Reply:
As discussed in "[__prevent compatibility regressions__](https://docs.python.org/3/howto/pyporting.html#prevent-compatibility-regressions)" this directive _changes the way that Python 2 operates_ to make legacy print statements a syntax error.  This prevents Python 2 developers from adding code that breaks Python 3 compatibility.
```python
python2 -c "from __future__ import print_function ; print 'Hi'"  # SyntaxError
```

Reply:
This is a good idea, it may make sense to standardize this as more people are shifting to Python 3.